parse/PythonParser.cpp

```diff
@@ -98,13 +98,20 @@ PythonParser::PythonParser(PythonCommon& _python, const boost::filesystem::path&
             .def(py::self_ns::self * double())
             .def(py::self_ns::self * py::self_ns::self)
             .def(double() * py::self_ns::self)
+            .def(py::self_ns::self / py::self_ns::self)
             .def(py::self_ns::self + int())
             .def(py::self_ns::self + double())
             .def(py::self_ns::self + py::self_ns::self)
             .def(py::self_ns::self - py::self_ns::self)
+            .def(int() + py::self_ns::self)
             .def(int() - py::self_ns::self)
+            .def(py::self_ns::self - int())
+            .def(double() <= py::self_ns::self)
```

Comment:
Is it a problem that, depending on context, `A <= B` could be a Condition or a ValueRef::Operation?

When matching conditions, it should treat `A <= B` as a condition, not a ValueRef(`A <= B`), even if it's used in the context `A <= B <= C`, which could otherwise be interpreted as a valueref whose result is then compared to C in a ValueTest condition.

But when specifically matching a ValueRef, then `A <= B` should be a ValueRef, unless in the context `A <= B <= C` in which case it should probably be a parse error to avoid ambiguity, even if the C++ Parser would accept it now.